### PR TITLE
ticket 11-sort articles

### PR DESCRIPTION
**Pull request by Gazdean**

### CODE REVIEW

**Review by nc-poonam-rajput — COMMENTED** [review]
image
Watch out for how tiny and out of place the menu is in mobile view

Bit picky but it was slightly jarring having created_at in the URL when the menu says date. I'd prefer both to say date really. 
image

File: src/Components/Articles.jsx [hunk 1]
```diff
@@ -39,34 +49,63 @@ export default function Articles() {
...
+            <option value="created_at">date</option>
+            <option value="comment_count">Comment count</option>
+            <option value="votes">Votes</option>
+          </select>
 
+          <label htmlFor="order">order: {}</label>
+          <select onChange={handleSelectOrder} id="order" >
+            <option value="desc">Descending</option>
+            <option value="asc">Ascending</option>
+          </select>
```
Comment (nc-poonam-rajput) [inline src/Components/Articles.jsx:106]: I reckon all the filtering code could be a sub-component to debloat this component code.

Comment (Gazdean) [reply src/Components/Articles.jsx:106]: done extra component
looks better now

Comment (Gazdean) [reply src/Components/Articles.jsx:106]: sorted date in both url and options

**Pull request merged**